app/sider/SendEndring.tsx

```diff
@@ -40,6 +41,13 @@ export default function SendEndringSide() {
     EFritekstFeil.MANGLER_TEKST,
   );
   const [erKnappTrykketPå, settKnappTrykketPå] = useState<boolean>(false);
+  const [erSamtykkeBekreftet] = useBekreftetSamtykke();
+
+  useEffect(() => {
+    if (!erSamtykkeBekreftet) {
+      navigate(hentPathForSteg(ESteg.FORSIDE));
```

Comment:
Hva skjer dersom du huker av for "jeg samtykker" og deretter fjerner avhukingen? Blir det en "refresh" eller skjer det ingenting? Tenker iallefall at man kanskje ikke skal redirecte hvis man allerede står på forsiden?